docs/configuration/images.md

```diff
@@ -16,4 +16,70 @@ FIXME: add content
+2. **Multi-Image**: Instruments like TARDIS have an image per detector. For these cases the next detector should be chosen before the image is loaded (although it is not neccessary, that can still be changed after) and the new template will not be applied until that new image is loaded.
+
+For either path toggling the selected detector before `Save detector boundary` has been selected will change the current template.
+
+*Note*: There is one special case at the moment - Image Plate 3 for TARDIS. The full boundary of the plate is provided to make it easier to align, but after it is saved you will see the boundary shrink. This is to further crop off useless data at the edges.
+
+![TARDIS IP3 Crop](img/TARDIS_IP3.gif)
+
+### Edit Detector Outline
+
+Depending on the image or prefered color map the default settings for the template may be hard to see, or you may want to differentiate between multiple detectors with different boundaries. For these needs there is the option to change the color, line style and thickness.
+
+![Template Style Options](img/template_style.gif)
+
+For the PXRDIP instrument the bounding box inputs will also be enabled. There is sometimes a need to edit the exact size of these templates and these inputs allow for that. At the moment this only applies to PXRDIP and only allows the boundaries to be made smaller than the defaults.
+
+### Finalize Instrument
+
+Each time that a template is saved its name is added to the `Finalize Instrument` section and the option to `Complete and Reload Instrument` is enabled.
+
+![Finalize LLNL Instrument Information](img/finalize_llnl.png)
+
+At any point in the workflow you can use the `Cancel LLNL Import` to clear the progress you've made and return everything to its defaults. This is the only way to re-enable instrument selection once the first image has been loaded.
+
+If everything seems correct you can select `Complete and Reload Instrument`. This will crop all of the images as well as build a new config file with updated row and column sizes as well as tilt values (if appropriate). The config that you selected at the very beginning will be used for this step and will just be updated internally where neccessary. This new configuration file as well as the newly cropped images will all be loaded into the GUI.
+
+![LLNL Save and Reload](img/save_and_reload.gif)
+
+*Note*: This save and reload process does **not** change any of the original files. You can save this new config through `File->Save->Configuration` and the images through `File->Save->Images`, or you can save everything as a state file with `File->Save->State`. For more detailed information on state files see the doc [here](state.md).
```

Comment:
```suggestion
*Note*: This does **not** modify your original configuration file. You can save this new config through `File->Save->Configuration` and the images through `File->Save->Images`, or you can save everything as a state file with `File->Save->State`. For more detailed information on state files see the doc [here](state.md).
```

I don't think the user needs to know that saving and reloading is happening underneath (as this is an implementation detail).